lib/browser/api/view.ts

```diff
@@ -1,3 +1,6 @@
+import { EventEmitter } from 'events';
 const { View } = process._linkedBinding('electron_browser_view');
 
+Object.setPrototypeOf((View as any).prototype, EventEmitter.prototype);
```

Comment:
Yes, it's because these classes derive from each other and gin::Wrappable doesn't support that, and EventEmitterMixin relies on gin::Wrappable (not gin_helper::Wrappable).